src/HMI/tmplits.css

```diff
@@ -132,4 +132,55 @@ input::-webkit-outer-spin-button,
 input::-webkit-inner-spin-button {
   -webkit-appearance: none;
   margin: 0;
+}
+
+.tmplit-table {
+    width: 100%;
+    display: grid;
+    padding: 0px 20px 20px 20px;
+    overflow: auto;
+    grid-template-columns: var(--tmplit-table-grid-template-columns, 1fr 4fr 1fr);
+    text-align: var(--tmplit-table-text-align, left);
+    grid-template-rows: var(--tmplit-table-grid-template-rows, repeat(auto-fill, 50px));
+    height: var(--tmplit-table-height, 470px);
+    background-color: var(--tmplit-table-background-color, var(--md-sys-color-surface-container, #FFFFFF));
+}
+
+.tmplit-table-header {
+    font-size: var(--tmplit-table-header-font-size, 24px);
+    font-weight: var(--tmplit-table-header-font-weight, 700);
+    padding: var(--tmplit-table-header-padding, 20px 20px 10px 20px);
+    text-align: var(--tmplit-table-header-text-align, center);
+    color: var(--tmplit-table-header-color, var(--md-sys-color-on-surface, black));
+    background-color: var(--tmplit-table-header-background-color, var(--md-sys-color-surface-container, #FFFFFF));
+}
+
+.tmplit-table-row {
+    display: grid;
+    grid-column: var(--tmplit-table-row-grid-column, 1 /10);
```

Comment:
This particular var has a challenging name, readability wise.
Any thoughts on how we might make it more clear? I don't have any great ideas, I think this is just a challenge with CSS vars of a certain specificity. Glad to see a description in the docs.

Reply:
Yeah i agree its not great for readability. But I couldn't think of a better way to add more hierarchy without breaking the css naming style. Kind of reminds me of twitter hashtags. #ihopeyoucanreadthis

Reply:
#yepiseewhatyouresaying